src/pages/academy/blog/index.tsx

```diff
@@ -69,6 +69,38 @@ const ArticlesPage = ({ data }) => {
         og_description:
             'Educational content at your fingertips – everything you need to know to start trading or upgrade your trading skills.',
     }
+
+    const [scrollPosition, setScrollPosition] = React.useState(0)
+    const handleScroll = () => {
+        const position = window.pageYOffset
+        setScrollPosition(position)
+    }
+
+    React.useEffect(() => {
+        window.addEventListener('scroll', handleScroll, { passive: true })
+
+        return () => {
+            window.removeEventListener('scroll', handleScroll)
+        }
+    }, [])
+
+    React.useEffect(() => {
+        scrollPosition && localStorage.setItem('scroll', scrollPosition.toString())
+    }, [scrollPosition])
+
+    React.useEffect(() => {
+        const prevScroll = parseInt(localStorage.getItem('scroll'))
```

Comment:
I think if `localStorage.getItem('scroll')` return `null` then becomes `NaN`. What do you think?

Reply:
You are right but I think it never returns `null` due to `useEffect` on line 79. What do you think?

Reply:
I think there are chances. In the first page load for example.
We can simple prevent hat by adding `prevScroll !== NaN` to the condition.
BTW I'm not sure 100%, just a suggestion.